functions.qmd

```diff
@@ -2,7 +2,7 @@
 
 ## Naming
 
-As well as following the general advice for [object names], strive to use verbs for function names:
+As well as following the general advice for object names in @sec-objectnames, strive to use verbs for function names:
```

Comment:
how does this reference work? I would have expected maybe `[object names](syntax#sec-objectnames)` instead.

Reply:
I've added {#sec-objectnames} accordingly, but linking is also good

Reply:
is `@` being resolved by quarto, or GitHub markdown? Any reference you could point me to?

Reply:
does this [Creating References](https://quarto.org/docs/books/book-crossrefs.html#creating-references) documentation answer your question?

Reply:
indeed! looks like a quarto thing then. thanks for the reference!